.pre-commit-config.yaml

```diff
@@ -115,7 +138,7 @@ repos:
           [
             "--count",
             "--select",
-            "E101,W191,E201,E202,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823",
+            "E101,E111,E112,E113,E201,E202,E30,E502,E722,E901,E902,E999,F401,F402,F404,F405,F406,F407,F501,F502,F503,F504,F505,F506,F507,F508,F509,F541,F701,F702,F703,F704,F706,F707,F822,F823,F901,W191,W291,W292,W293,W391",
```

Comment:
Would an exclude list be shorter?

Reply:
🤞 it soon will be. There's a fair number of https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

Reply:
Done! It does simplify things considerably. Thanks for the suggestion.